storage/Devices/DeviceImpl.cc

```diff
@@ -140,6 +140,40 @@ namespace storage
     void
     Device::Impl::add_modify_actions(Actiongraph::Impl& actiongraph, const Device* lhs) const
     {
+	add_reallot_actions(actiongraph, lhs);
+    }
+
+
+    void
+    Device::Impl::add_reallot_actions(Actiongraph::Impl& actiongraph, const Device* lhs) const
```

Comment:
Some comments would be very nice in this function. I have no idea what it does; it's not obvious from its name, and the purpose of those set operations does not become clear from reading the code.